cves/kernel/CVE-2021-45095.yml

```diff
@@ -173,8 +185,9 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: Most likely not, as this was simply an error on the developer's side which
+        resulted in them forgetting to include it.
```

Comment:
> forgetting to include it

Forgetting to include what? I see in the commit they forgot to put the `__sock_put()` method so I'm assuming that is "it"

Reply:
I have updated the file to include more details on this.